app/views/support_interface/guidance/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<% content_for :title, t('page_titles.guidance') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <p class='govuk-body'>Within the support console there is sensitive information and personal data about the users of the Apply for teacher training service which we store inline with our Data Protection Agreement and Privacy Notice.</p>
+
+    <p class='govuk-body'>As a user of the Apply support console there are specific things you should be aware of and practices to ensure you use data in the appropriate manner and we are not at risk of mis-use or breach of GDPR and other data security policies.</p>
+
+    <p class='govuk-body'>Most practice in line with GDPR is common sense but the key messages are:
+      <ul class='govuk-list govuk-list--bullet'>
+        <li>only access what data you need to</li>
+        <li>think whether there is a reason to view what you’re accessing</li>
+        <li>don't copy data and store in other locations</li>
```

Comment:
Actually given #2698, in this particular instance, use `do not`, not `don’t`.

Reply:
Yea good point 👍 i def shouldn't be blindly copy and pasting from the doc. My bad.